src/lib/subname-helpers.ts

```diff
@@ -1,16 +1,17 @@
+import { bytesToPacket } from "@ensdomains/ensjs/utils";
 import { type Hex, concat, keccak256, namehash, toHex } from "viem";
 
 // TODO: pull from ens utils lib or something
 export const ROOT_NODE = namehash("");
+export const ETH_NODE = namehash("eth");
 
-// TODO: this should probably be a part of some ens util lib
+// TODO: this should probably be a part of some ens util lib but isn't exposed by viem or ensjs
 export const makeSubnodeNamehash = (node: Hex, label: Hex) => keccak256(concat([node, label]));
 
-// https://github.com/wevm/viem/blob/main/src/utils/ens/encodeLabelhash.ts
-export const encodeLabelhash = (hash: Hex): `[${string}]` => `[${hash.slice(2)}]`;
-
+// converts uint256-encoded nodes to hex
 export const tokenIdToLabel = (tokenId: bigint) => toHex(tokenId, { size: 32 });
 
+// TODO: this should be part of some more standard ens helper lib
 // https://github.com/ensdomains/ens-subgraph/blob/master/src/utils.ts#L68
 const INVALID_CHARACTER_CODES_IN_LABEL = new Set([0, 46, 91, 93]);
```

Comment:
```suggestion
const UNINDEXABLE_CHARACTER_CODES_IN_LABEL = new Set([0, 46, 91, 93]);
```

It's best we not use the word "invalid" here because that can be interpreted in multiple ways. For example, if invalid means not compliant with ENSIP-15 then there's far more than 4. Therefore suggesting this more specific terminology.

@shrugs Also suggest creating distinct named constants for each of these 4 character codes so that the logic here is more self documenting :+1: